### ISSUE TITLE
Restyling the build details page

### ISSUE THREAD

**Issue by pradyunsg** [readthedocs/readthedocs.org#7666]
**Description**

I've had a bit of CSS I inject into the builds page for ReadTheDocs locally to make them nicer. And well... I think it's nice+stable enough by now, that it might be worth upstreaming this content.

<details>
<summary>Click to see the CSS</summary>

```
#build-commands {
    padding: 3px 0;
    border-radius: 3px;
    background: #2f333d;
}

div.build-command {
    border: none;
    border-radius: 0;
    background: transparent;
    color: white;
    padding: 0;
    margin: 0;
}
div.build-command-successful .build-command-run span:hover {
    background: #282c34;
}
div.build-command-failed .build-command-run span:hover {
    background: #944;
}
div.build-command-run {
    margin: 0;
    padding: 0;
    border-radius: 0;
}
div.build-command-run span {
    margin: 0;
    padding: 0.25em 0.5em;
}
div.build-command-run span::before {
    content: "$ ";
}
div.build-command div.build-command-run span,
div.build-command div.build-command-output span {
    white-space: pre-wrap;
}
div.build-command div.build-command-output {
    background: #1d1f23;
    margin: 0;
}
div.build-command-output span {
    white-space: pre-wrap;
}

div.build-command div.build-command-meta {
    background: #21252b;
}
```

</details>

**Screenshots**

Regular output, with one element "expanded" and hovering on the `git clean` statement

(current)

![Screenshot 2020-11-15 at 2 58 45 AM](https://user-images.githubusercontent.com/3275593/99157403-8166c500-26ee-11eb-8149-a62f77bc6ffa.png)

(proposed)

![Screenshot 2020-11-15 at 2 56 10 AM](https://user-images.githubusercontent.com/3275593/99157374-2d5be080-26ee-11eb-9aa0-d1c47182ccc0.png)

When there's a failure

(current)

![Screenshot 2020-11-15 at 2 59 14 AM](https://user-images.githubusercontent.com/3275593/99157406-8fb4e100-26ee-11eb-9200-9903ac5254d0.png)

(proposed)

![Screenshot 2020-11-15 at 2 57 00 AM](https://user-images.githubusercontent.com/3275593/99157380-41074700-26ee-11eb-9185-708fc5ae7b6a.png)

**Additional context**

Relevant CSS lives here: https://github.com/readthedocs/readthedocs.org/blob/43073b15999d18f96532682ba838717521119abf/readthedocs/builds/static/builds/css/detail.css

**Comment by pradyunsg** [comment]
Notably, this doesn't touch any of the markup, which means it should "just work" with the existing HTML -- the only thing to change is the CSS file linked at the end. :)

I won't be able to make a PR anytime soon, but hey, this change is nice enough that I feel like not sharing it would be bad. :)

**Comment by humitos** [comment]
Thanks @pradyunsg! We have been working on a redesign of the whole site. @agjohnson was leading this and did all of the work. I'm pinging him here in case he can grab some ideas from here and mix them with what he already have.

I hope we can deploy the beta state of the new theme so people can start testing it and suggesting improvements!

**Comment by agjohnson** [comment]
@pradyunsg Thanks for sharing!

The good news: the new theme design ends up with a build detail page that looks pretty similar. There are a lot of new bonus features too: per line linking, section linking, and escape code highlighting!

The bad news: the new theme is a 100% rewrite of all of the pages and none of the existing css or html structure is reused. It's also a huge project and we aren't yet to private/public beta testing.

So, we're anxious to start testing, but aren't quite there yet. I don't recommend anyone focuses much on our CSS or really even our templates in the meantime, as this work will most likely be lost or not usable with the new theme.

@pradyunsg it sounds like we should include you in the private beta when we're ready. If you'd like, we can reach out with details when we have something ready to test more. We'll have a lot of QA to do and feedback would be extremely helpful here.

**Comment by pradyunsg** [comment]
> It sounds like we should include you in the private beta when we're ready.

Please do reach out!

> I don't recommend anyone focuses much on our CSS or really even our templates in the meantime, as this work will most likely be lost or not usable with the new theme.

It would be good to add this context to the README of this project. :)

---

FWIW, I do think this CSS would be nice to have (as in, in the current non-rewrite variant, since it's basically a single SCSS file getting changes) -- while the rewrite is under process. OTOH, I am already injecting this CSS, so... it doesn't really affect me. ;)

**Comment by pradyunsg** [comment]
Closing since I don't think this needs to be tracked.

**Comment by pradyunsg** [comment]
I'll flag that it's almost midway through 2022 -- and this was rejected since there is a new theme under development. Have there been any public updates on the new theme's rollout?

If it's not coming out in the order of weeks, may I suggest adding the ~50 lines of CSS that the issue description has? There have been no changes to the UI for over an year now, and this is a very quick win that will have no ongoing costs.

**Comment by ericholscher** [comment]
@pradyunsg Sorry for this, and I appreciate your frustration. We're taking some steps to prioritize getting the beta dashboard out, and I agree it's an unfortunate wait on that. I really hope we have some more things to show there in the next couple weeks. I'm not going to push forward with this change quite yet, but hopefully within 2-4 weeks we'll have a link to share with you to help us beta test the new dashboard.
